prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -116,11 +116,14 @@ public void onRemove(Entity entity) throws FalconException{
         if (entity.getEntityType() != EntityType.PROCESS){
```

Comment:
The check here could cause issues. Entity can be cluster or datasource. In which case, the explicit cast below will cause issues.